game-app/src/app/components/Gameboard/Gameboard.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import Image from "next/image";
+
+export default function Gameboard() {
+  return (
+    // TODO: Extract colors to the Tailwind theme once properly defined
```

Comment:
can you create an issue for that plz

Reply:
done https://github.com/keep-starknet-strange/tsubasa/issues/41